classes/PodsInit.php

```diff
@@ -574,10 +571,10 @@ public function register_assets() {
 					'jquery',
 					'pods-i18n',
 				),
-				'1.0.1',
+				'1.0.8',
 				true
 			);
-			wp_register_script( 'pods-select2', $select2_i18n, array( 'pods-select2-core' ), '1.0.1', true );
+			wp_register_script( 'pods-select2', $select2_i18n, array( 'pods-select2-core' ), '1.0.8', true );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 74b1b1c06020dc630e610d3d21cee18d</sub>